bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,514 @@
+
+| Parameter      |Description                                                       |Mandatory|Default                                         |
+|----------------|------------------------------------------------------------------|---------|------------------------------------------------|
+| defaultUserId  |Default user id for HTTP authentication when not set in the Thing |    no   |admin                                           |
+| defaultPassword|Default password for HTTP authentication when not set in the Thing|    no   |admin                                           |
+
+## Thing Configuration
+
+|Parameter         |Description                                                   |Mandatory|Default                                           |
+|------------------|--------------------------------------------------------------|---------|--------------------------------------------------|
+|deviceIp          |IP address of the Shelly device                               |    yes  |none                                              |
+|userId            |The user id used for HTTP authentication                      |    no   |none                                              |
+|password          |Password for HTTP authentication*                             |    no   |none                                              |
+|lowBattery        |Threshold for battery level. Set alert when level is below.   |    no   |20 (=20%), only for battery powered devices       |
+|updateInterval    |Interval for the background status check in seconds.          |    no   |1h for battery powered devices, 60s for all others|
+|eventsButton      |true: register event "trigger when a button is pushed"        |    no   |false                                             |
+|eventsPush        |true: register event "trigger on short and long push"         |    no   |false                                             |
+|eventsSwitch      |true: register event "trigger of switching the relay output"  |    no   |true                                              |
+|eventsSensorReport|true: register event "posted updated sensor data"             |    no   |true for sensor devices                           |
+|eventsCoIoT       |true: Listen for CoIoT/COAP events                            |    no   |true for battery devices, false for others        |
+
+Independent from the updateInterval setting the binding will perform a settings refresh from the device once per minute.
+There is no event from the device that the settings have be changed by the Shelly App so this is kind of a fallback to get updates at least once per minute. 
+
+
+
+## Channels
+
+### General Notes
+
+- Various channels are only visible for linking when you click Show More in the channel definition
+- input channel is only available with firmware > 1.5.6, otherwise NULL
+- use channel rollerpos only if you need the inverted value, otherwise use the control channel and Item Type Number, see example
```

Comment:
```suggestion
- Use the channel `rollerpos` only if you need the inverted value, otherwise use the control channel with item type `Number`, see example below.
```

Reply:
done